search/search_test.go

```diff
@@ -400,6 +400,46 @@ func Test_SearchForRefs(t *testing.T) {
 	require.Equal(t, want[0].Path, got[0].Path)
 }
 
+func Test_truncateLine(t *testing.T) {
+	tests := []struct {
+		name         string
+		line         string
+		maxCharCount int
+		want         string
+	}{
+		{
+			name: "line equal to max character count returns line",
+			// len(line) = 8
```

Comment:
Everything else looks good to me, can we just remove these commented lines if not needed?

Reply:
i was keeping them in for reference about the diff between line length and rune length - what if i consolidate on one line?